packages/client/src/pages/LoginPage.tsx

```diff
@@ -279,8 +281,28 @@ export const LoginPage = observer(() => {
             PASSWORD_CONFIRMATION: '',
         },
     });
+    useEffect(() => {
+        fetchLoginTypesAllowed();
+    }, []);
 
-    const location = useLocation();
+    /**
+     * Find which types of logins are available to the user.
+     */
+    const fetchLoginTypesAllowed = async () => {
```

Comment:
`configStore` should already have this information. Another network call should not be necessary